query-planner-js/src/buildPlan.ts

```diff
@@ -1051,6 +1070,10 @@ class FetchGroup {
       return false;
     }
 
+    if (this.isKnownUseful) {
```

Comment:
I'd just as this to the above clause on line 1069.

Reply:
Done.